music21/scale/__init__.py

```diff
@@ -893,7 +904,58 @@ def buildNetwork(self):
             pitchSimplification=None)
         self._net.fillMelodicMinor()
 
+    def getRelativeMajor(self):
+        return MajorScale(self.tonic)
 
+    def getRelativeMinor(self):
+        return MinorScale(self.tonic)
```

Comment:
this also gets the parallels.  But minor scales shouldn't getRelativeMinor anyhow -- it doesn't make sense for a minor to have a parallel minor or a relative minor.  They already are a form of minor.

Reply:
> But minor scales shouldn't getRelativeMinor anyhow -- it doesn't make sense for a minor to have a parallel minor or a relative minor. They already are a form of minor.

I respectfully disagree on this point, as I think there are edge cases in which it would be helpful for users to be able to getRelativeMinor/getParallelMinor when they need to determine a key signature or simply not receive an error message when using these functions. 
Also, the basic MajorScale class, for example, allows getRelativeMajor much in the same way as the basic MinorScale class allows getRelativeMinor. For the sake of consistency, I think it is important to have these functions work for the non-natural minor scales as well. Or, perhaps, would you say that these other scales should not have such functionality either?

Nevertheless, I greatly appreciate the feedback and plan to continue working on making the improvements that you and jacobtylerwalls have mentioned sometime in the near future.

Reply:
> there are edge cases in which it would be helpful for users to be able to getRelativeMinor/getParallelMinor when they need to determine a key signature or simply not receive an error message when using these functions.

I agree, we shouldn't bother trying to remove any of this functionality, it should just work as currently documented, and you have demonstrated it doesn't work for melodic/harmonic/jazz minor. I think this is neither here nor there, though; we should revert the new function definitions. Your patch works with just setting the attributes for relativeMajorDegree and relativeMinorDegree, nothing else needed.